data-prepper-plugins/csv-processor/src/main/java/org/opensearch/dataprepper/plugins/csvinputcodec/CsvCodecConfig.java

```diff
@@ -12,7 +12,7 @@
 import java.util.Objects;
 
 /**
- * Configuration class for {@link CsvCodec}.
+ * Configuration class for {@link CsvInputCodec}.
  */
 public class CsvCodecConfig {
```

Comment:
Please rename this class to `CsvInputCodecConfig`. There might be some changes between this an output codec.

Reply:
I think this should be in the `csv-processor`. We could reduce code duplication by keeping these CSV plugins together. I don't see any benefit to end users to split these out - since they share the same dependencies.